src/index.css

```diff
@@ -1,5 +1,40 @@
+@value darkerGray from "./colors.css";
+
 body {
   margin: 0;
   padding: 0;
-  font-family: sans-serif;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+}
+
+h1 {
```

Comment:
Not that it matters too much, but do we not want these 3 in the CSS file for the Header component?

Reply:
They shouldn't be in the Header component because if someone uses `h1` without the component it should still have the same style.

Reply:
That's actually exactly what I was wondering about.
I would imagine generally no-one should be using raw tags, but if they do use e.g. a raw h1 header, bypassing our component, that would mean they don't want the standardization from the style guide. Just like if someone uses a raw input tag, bypassing our Input, they wouldn't get any of our styles for that.
But, I'm not too sure either way.